javafx-archetype-fxml/src/main/resources/archetype-resources/pom.xml

```diff
@@ -36,7 +34,7 @@
             <plugin>
                 <groupId>org.openjfx</groupId>
                 <artifactId>javafx-maven-plugin</artifactId>
-                <version>${javafx.maven.plugin.version}</version>
+                <version>${javafx-maven-plugin-version}</version>
```

Comment:
By the way, the `<executions>` block is not required, both goals are applied

Reply:
Done!